docs_user/assemblies/assembly_migrating-ceph-rbd.adoc

```diff
@@ -4,13 +4,40 @@
 
 = Migrating Red Hat Ceph Storage RBD to external RHEL nodes
 
-For hyperconverged infrastructure (HCI) or dedicated Storage nodes that are running {Ceph} version 6 or later, you must migrate the daemons that are included in the {rhos_prev_long} control plane into the existing external Red Hat Enterprise Linux (RHEL) nodes. The external RHEL nodes typically include the Compute nodes for an HCI environment or dedicated storage nodes.
+For hyperconverged infrastructure (HCI) or dedicated Storage nodes that are
+running {Ceph} version 6 or later, you must migrate the daemons that are
+included in the {rhos_prev_long} control plane into the existing external Red
+Hat Enterprise Linux (RHEL) nodes. The external RHEL nodes typically include
+the Compute nodes for an HCI environment or dedicated storage nodes.
 
-To migrate Red Hat Ceph Storage Rados Block Device (RBD), your environment must meet the following requirements:
+To migrate Red Hat Ceph Storage Rados Block Device (RBD), your environment must
+meet the following requirements:
 
 * {Ceph} is running version 6 or later and is managed by cephadm/orchestrator.
```

Comment:
```suggestion
* {Ceph} is running version 6 or later and is managed by cephadm/orchestrator.
```
Instead of "cephadm/ceph orchestrator", is it accurate to say " ...is managed by the Ceph Orchestrator with Cephadm utility"?

Reply:
I'd just say "managed by Cephadm" to keep things simple and use wording that I'm pretty sure won't confuse customers.
I'll change it to "managed by Cephadm" if it's ok with you.